Sources/Xcodeproj/pbxproj().swift

```diff
@@ -139,7 +139,9 @@ func xcodeProject(
 
     // Also set the `Xcode` build preset in Swift to let code conditionalize on
     // being built in Xcode.
-    projectSettings.common.OTHER_SWIFT_FLAGS += ["-DXcode"]
+    projectSettings.common.OTHER_SWIFT_FLAGS = ["$(inherited)"]
+        + (projectSettings.common.OTHER_SWIFT_FLAGS ?? [])
+        + ["-DXcode"]
```

Comment:
We can move this line before we append the additional compiler flags to simplify this to `projectSettings.common.OTHER_SWIFT_FLAGS = ["$(inherited)", "-DXcode"]`

Reply:
I had considered this but decided against it because I didn't know if the order of settings needed to be maintained. 

For example, making the change indicated would have the effect of moving "-DXcode" to no longer be at the end, in the case that options.flags.swiftCompilerFlags isn't empty.  Additionally, if the current or future implementation of Xcode.Project() were to set OTHER_SWIFT_FLAGS, for some reason, those would be lost with this change.

While I know that the position of -DXcode probably doesn't matter and the Xcode.Project() probably won't set OTHER_SWIFT_FLAGS, I felt the code I wrote best preserved the existing semantics and protected against these small possible outlier issues I described.

Reply:
We usually start with assigning a value to these arrays and then use `+=` to add more content into them. In this case, the order doesn't really matter and it'll be more confusing that we're not appending in later calls.

Reply:
Thanks for the feedback on this.